spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'Tests for User model validation ' do
+    subject { User.new(name: 'Rito', email: 'abcd@gmail.com', password: '123456') }
+
+    before { subject.save }
+
+    it 'name should be present' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'name should not be blank' do
+      subject.name = ''
+      expect(subject).to_not be_valid

```

Comment:
- [ ] Well done writing your Rspec tests so far. However, there are a few tests that haven't passed please fix them.
![Git-Bksh__c_Users_mbakr_OneDrive_Desktop_Junk_Receipe-app-rails 10_15_2022 1_20_52 AM](https://user-images.githubusercontent.com/47625407/195957045-69b3b101-2d1b-4748-956b-a102212419ad.png)

Reply:
Ah!! Sure, will fix it

Reply:
Ah!! Sure, will fix it